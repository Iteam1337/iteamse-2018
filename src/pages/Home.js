// @flow

import * as React from 'react'
import * as Iteam from '../typings/iteam.flow'
import * as IteamCMS from './__generated__/HomePage'
import { Query } from 'react-apollo'
import gql from 'graphql-tag'
import iteamStylized from './img/iteam_stylized.png'
import Header from '../components/Header/Header'
import Team from '../components/Team/Team'
import GridColumn from '../components/Grid/GridColumn'
import PaddedRow from '../components/Grid/PaddedRow'
import OperationsBanner from '../components/Banners/OperationsBanner'
import H1 from '../components/Typography/H1'
import Paragraph from '../components/Typography/Paragraph'
import Link from '../components/Link/Link'
import styled from 'styled-components'
import MediaQuery from 'react-responsive'
import { media } from '../theme'

type Props = Iteam.ApolloBase<IteamCMS.HomePage>

export const HomePageQuery = gql`
  query HomePage {
    pageStart {
      headerImage
      headerText1
      headerText2
      headerTextBgColor
      codeMobileImage
      codeText
      codeTitle
      codeLogo
      codeSlug
      codeLinkText
      cultureMobileImage
      cultureText
      cultureTitle
      cultureLogo
      cultureSlug
      cultureLinkText
      strategyMobileImage
      strategyText
      strategyTitle
      strategyLogo
      strategySlug
      strategyLinkText
    }
  }
`

const HomeGridColumn = GridColumn.extend`
  background-color: ${({ theme }) => theme.colors.concrete};
`

const Content = styled.div`
  display: grid;
  grid-column-gap: 40px;
  grid-template-columns: 1fr;
  padding-bottom: 40px;
  padding-top: 40px;

  ${media.desktop`
    grid-template-columns: 550px 1fr;
    padding-bottom: 100px;
    padding-top: 100px;
  `};
`

const Texts = styled.div`
  align-items: center;
  display: grid;
  grid-row-gap: 40px;
`

const TextBlock = PaddedRow.extend`
  &:not(:last-child) {
    margin-bottom: 40px;
  }

  ${media.desktop`
    &:not(:last-child) {
      margin-bottom: 0;
    }
  `};
`

const StylizedIteam = styled.img`
  display: none;

  ${media.desktop`
    display: block;
  `};
`

const CompanyLogo = styled.img`
  display: block;
  margin-bottom: 20px;
`

const MobileImageWrap = styled.div`
  align-items: center;
  display: flex;
  justify-content: center;
`

const MobileImage = styled.img`
  margin-bottom: 20px;
  width: 50%;
`

const Home = () => {
  return (
    <Query query={HomePageQuery}>
      {({ loading, data: { pageStart } }: Props) => {
        if (loading) {
          return null
        }

        return (
          <React.Fragment>
            <Header
              backgroundImage={pageStart.headerImage}
              messageBgColor={pageStart.headerTextBgColor}
              messageOne={pageStart.headerText1}
              messageTwo={pageStart.headerText2}
            />

            <HomeGridColumn>
              <Content>
                <StylizedIteam src={iteamStylized} />
                <Texts>
                  <TextBlock>
                    <MediaQuery maxDeviceWidth={480}>
                      <MobileImageWrap>
                        <MobileImage src={pageStart.codeMobileImage} />
                      </MobileImageWrap>
                    </MediaQuery>
                    <H1>{pageStart.codeTitle}</H1>
                    <Paragraph>{pageStart.codeText}</Paragraph>
                    <CompanyLogo src={pageStart.codeLogo} />
                    <Link to={`/case/${pageStart.codeSlug}`}>
                      {pageStart.codeLinkText}
                    </Link>
                  </TextBlock>

                  <TextBlock>
                    <MediaQuery maxDeviceWidth={480}>
                      <MobileImageWrap>
                        <MobileImage src={pageStart.strategyMobileImage} />
                      </MobileImageWrap>
                    </MediaQuery>
                    <H1>{pageStart.strategyTitle}</H1>
                    <Paragraph>{pageStart.strategyText}</Paragraph>{' '}
                    <CompanyLogo src={pageStart.strategyLogo} />
                    <Link to={`/case/${pageStart.strategySlug}`}>
                      {pageStart.strategyLinkText}
                    </Link>
                  </TextBlock>

                  <TextBlock>
                    <MediaQuery maxDeviceWidth={480}>
                      <MobileImageWrap>
                        <MobileImage src={pageStart.cultureMobileImage} />
                      </MobileImageWrap>
                    </MediaQuery>
                    <H1>{pageStart.cultureTitle}</H1>
                    <Paragraph>{pageStart.cultureText}</Paragraph>{' '}
                    <CompanyLogo src={pageStart.cultureLogo} />
                    <Link to={`/case/${pageStart.cultureSlug}`}>
                      {pageStart.cultureLinkText}
                    </Link>
                  </TextBlock>
                </Texts>
              </Content>
            </HomeGridColumn>

            <OperationsBanner />
            <Team shortName={['met', 'rln', 'msr', 'jmn']} />
          </React.Fragment>
        )
      }}
    </Query>
  )
}

export default Home
