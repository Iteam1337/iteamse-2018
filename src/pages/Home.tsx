import gql from 'graphql-tag'
import * as React from 'react'
import { Query } from 'react-apollo'
import Helmet from 'react-helmet-async'
import { StartQuery } from '../../typings/iteamse'
import Block from '../components/Blocks/Block'
import ImageBlock from '../components/Blocks/ImageBlock'
import GridColumn from '../components/Grid/GridColumn'
import Team from '../components/Team/Team'
import howWeWorkImage from './img/illustrations-group-3.svg'

export const HOME_PAGE_QUERY = gql`
  query Start {
    pageStart2019 {
      team
      weAreTitle
      weAreText
      weOfferTitle
      weOfferText
      ourMethodTitle
      ourMethodText
    }
  }
`

class HomeQuery extends Query<StartQuery> {}

const Home = () => {
  return (
    <>
      <Helmet>
        <title>Iteam - There's a better way</title>
        <meta property="og:title" content="Iteam - There's a better way" />
        <meta property="twitter:title" content="Iteam - There's a better way" />
      </Helmet>

      <HomeQuery query={HOME_PAGE_QUERY}>
        {({ loading, data }) => {
          if (loading || !data) {
            return null
          }

          const { pageStart2019 } = data

          return (
            <>
              <GridColumn>
                <Block title={pageStart2019.weAreTitle}>
                  {pageStart2019.weAreText}
                </Block>
                <Block title={pageStart2019.weOfferTitle}>
                  {pageStart2019.weOfferText}
                </Block>
                <ImageBlock image={howWeWorkImage} />
                <Block title={pageStart2019.ourMethodTitle}>
                  {pageStart2019.ourMethodText}
                </Block>
              </GridColumn>
              <Team shortName={pageStart2019.team} />
            </>
          )
        }}
      </HomeQuery>
    </>
  )
}

export default Home
