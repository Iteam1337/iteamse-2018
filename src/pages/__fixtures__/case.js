export const workCase = {
  headerBgColor: '#1616B2',
  logo:
    '//images.ctfassets.net/rj4r6yfcesw5/4p0ptdLj9SaECekOI4c6C2/c2cdeab1cebc9650ebb0e7778357c231/AF_ordbild_vit.png',
  slug: 'arbetsformedlingen',
  thumbnailImage:
    '//images.ctfassets.net/rj4r6yfcesw5/3Y7zZ25WbeOCU8w84kAEos/b97ff1899cf6f2e4dca0eb2b4f0d5a44/af.png',
  tags: ['API', 'Docker', 'UX', 'React', 'Vue'],
  title: 'Arbetsförmedlingen',
  introduction:
    'Aeneanurutrum, erat at tempus porttitor, mi eros vestibulum magna, eget semper mauris ipsum a nisi. Interdum et malesuada fames ac ante ipsum primis in faucibus. Class aptent taciti sociosqu ad litora torquent per conubia nostra, per inceptos himenaeos. Cras quis odio vel turpis pellentesque finibus. Pellentesque cursus diam ac ligula placerat volutpat. Vivamus dictum augue eu rutrum efficitur. Suspendisse ut pellentesque ante. Suspendisse odio velit, fringilla eu porta vel, elementum ac libero. Nam eu vestibulum magna. Vestibulum quis risus elementum, lobortis sapien at, rhoncus enim. Nam accumsan odio lacus, at facilisis lacus rhoncus vel. Suspendisse sollicitudin a ipsum at cursus.',
  introductionTitle: 'En ny digital tjänst',
  process:
    'Aeneanurutrum, erat at tempus porttitor, mi eros vestibulum magna, eget semper mauris ipsum a nisi. Interdum et malesuada fames ac ante ipsum primis in faucibus. Class aptent taciti sociosqu ad litora torquent per conubia nostra, per inceptos himenaeos. Cras quis odio vel turpis pellentesque finibus. Pellentesque cursus diam ac ligula placerat volutpat. Vivamus dictum augue eu rutrum efficitur. Suspendisse ut pellentesque ante. Suspendisse odio velit, fringilla eu porta vel, elementum ac libero. Nam eu vestibulum magna. Vestibulum quis risus elementum, lobortis sapien at, rhoncus enim. Nam accumsan odio lacus, at facilisis lacus rhoncus vel. Suspendisse sollicitudin a ipsum at cursus.',
  processTitle: 'Process',
  development:
    'Aeneanurutrum, erat at tempus porttitor, mi eros vestibulum magna, eget semper mauris ipsum a nisi. Interdum et malesuada fames ac ante ipsum primis in faucibus. Class aptent taciti sociosqu ad litora torquent per conubia nostra, per inceptos himenaeos. Cras quis odio vel turpis pellentesque finibus. Pellentesque cursus diam ac ligula placerat volutpat. Vivamus dictum augue eu rutrum efficitur. Suspendisse ut pellentesque ante. Suspendisse odio velit, fringilla eu porta vel, elementum ac libero. Nam eu vestibulum magna. Vestibulum quis risus elementum, lobortis sapien at, rhoncus enim. Nam accumsan odio lacus, at facilisis lacus rhoncus vel. Suspendisse sollicitudin a ipsum at cursus.',
  developmentTitle: 'Utveckling',
  aboutCompany:
    'Aeneanurutrum, erat at tempus porttitor, mi eros vestibulum magna, eget semper mauris ipsum a nisi. Interdum et malesuada fames ac ante ipsum primis in faucibus. Class aptent taciti sociosqu ad litora torquent per conubia nostra, per inceptos himenaeos. Cras quis odio vel turpis pellentesque finibus. Pellentesque cursus diam ac ligula placerat volutpat. Vivamus dictum augue eu rutrum efficitur. Suspendisse ut pellentesque ante. Suspendisse odio velit, fringilla eu porta vel, elementum ac libero. Nam eu vestibulum magna. Vestibulum quis risus elementum, lobortis sapien at, rhoncus enim. Nam accumsan odio lacus, at facilisis lacus rhoncus vel. Suspendisse sollicitudin a ipsum at cursus.',
  aboutCompanyTitle: 'Arbetsförmedlingen',
  partners: '[Digitalbyrån](https://hej.com)',
  partnersTitle: 'Samarbetspartners',
  contact:
    '- [Hans Rollman](mailto:hans.rollman@iteam.se)\n- Johan Littorin, Telenor, 0734 41 72 85',
  contactTitle: 'Presskontakt',
  quote:
    '"We feel that our crew at Iteam are more colleagues than suppliers. Rather than specifying a tech spec to work on, we collaborate, carefully chiseling out functionality as we go along. This way of working has minimized waste and it has also taken us further than we ever imagined”',
  quotePerson: 'Anna Andersson, chef Arbetsförmedlingen',
  __typename: 'Case',
}
