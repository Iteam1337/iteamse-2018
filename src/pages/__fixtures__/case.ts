export const workCase = {
  __typename: 'Case',
  aboutCompany:
    'Aeneanurutrum, erat at tempus porttitor, mi eros vestibulum magna, eget semper mauris ipsum a nisi. Interdum et malesuada fames ac ante ipsum primis in faucibus. Class aptent taciti sociosqu ad litora torquent per conubia nostra, per inceptos himenaeos. Cras quis odio vel turpis pellentesque finibus. Pellentesque cursus diam ac ligula placerat volutpat. Vivamus dictum augue eu rutrum efficitur. Suspendisse ut pellentesque ante. Suspendisse odio velit, fringilla eu porta vel, elementum ac libero. Nam eu vestibulum magna. Vestibulum quis risus elementum, lobortis sapien at, rhoncus enim. Nam accumsan odio lacus, at facilisis lacus rhoncus vel. Suspendisse sollicitudin a ipsum at cursus.',
  aboutCompanyTitle: 'Arbetsförmedlingen',
  contact:
    '- [Hans Rollman](mailto:hans.rollman@iteam.se)\n- Johan Littorin, Telenor, 0734 41 72 85',
  contactTitle: 'Presskontakt',
  development:
    'Aeneanurutrum, erat at tempus porttitor, mi eros vestibulum magna, eget semper mauris ipsum a nisi. Interdum et malesuada fames ac ante ipsum primis in faucibus. Class aptent taciti sociosqu ad litora torquent per conubia nostra, per inceptos himenaeos. Cras quis odio vel turpis pellentesque finibus. Pellentesque cursus diam ac ligula placerat volutpat. Vivamus dictum augue eu rutrum efficitur. Suspendisse ut pellentesque ante. Suspendisse odio velit, fringilla eu porta vel, elementum ac libero. Nam eu vestibulum magna. Vestibulum quis risus elementum, lobortis sapien at, rhoncus enim. Nam accumsan odio lacus, at facilisis lacus rhoncus vel. Suspendisse sollicitudin a ipsum at cursus.',
  developmentTitle: 'Utveckling',
  headerBgColor: '#1616B2',
  introduction:
    'Aeneanurutrum, erat at tempus porttitor, mi eros vestibulum magna, eget semper mauris ipsum a nisi. Interdum et malesuada fames ac ante ipsum primis in faucibus. Class aptent taciti sociosqu ad litora torquent per conubia nostra, per inceptos himenaeos. Cras quis odio vel turpis pellentesque finibus. Pellentesque cursus diam ac ligula placerat volutpat. Vivamus dictum augue eu rutrum efficitur. Suspendisse ut pellentesque ante. Suspendisse odio velit, fringilla eu porta vel, elementum ac libero. Nam eu vestibulum magna. Vestibulum quis risus elementum, lobortis sapien at, rhoncus enim. Nam accumsan odio lacus, at facilisis lacus rhoncus vel. Suspendisse sollicitudin a ipsum at cursus.',
  introductionTitle: 'En ny digital tjänst',
  logo:
    '//images.ctfassets.net/rj4r6yfcesw5/4p0ptdLj9SaECekOI4c6C2/c2cdeab1cebc9650ebb0e7778357c231/AF_ordbild_vit.png',
  partners: '[Digitalbyrån](https://hej.com)',
  partnersTitle: 'Samarbetspartners',
  process:
    'Aeneanurutrum, erat at tempus porttitor, mi eros vestibulum magna, eget semper mauris ipsum a nisi. Interdum et malesuada fames ac ante ipsum primis in faucibus. Class aptent taciti sociosqu ad litora torquent per conubia nostra, per inceptos himenaeos. Cras quis odio vel turpis pellentesque finibus. Pellentesque cursus diam ac ligula placerat volutpat. Vivamus dictum augue eu rutrum efficitur. Suspendisse ut pellentesque ante. Suspendisse odio velit, fringilla eu porta vel, elementum ac libero. Nam eu vestibulum magna. Vestibulum quis risus elementum, lobortis sapien at, rhoncus enim. Nam accumsan odio lacus, at facilisis lacus rhoncus vel. Suspendisse sollicitudin a ipsum at cursus.',
  processTitle: 'Process',
  quote:
    '"We feel that our crew at Iteam are more colleagues than suppliers. Rather than specifying a tech spec to work on, we collaborate, carefully chiseling out functionality as we go along. This way of working has minimized waste and it has also taken us further than we ever imagined”',
  quotePerson: 'Anna Andersson, chef Arbetsförmedlingen',
  slug: 'arbetsformedlingen',
  tags: ['API', 'Docker', 'UX', 'React', 'Vue'],
  team: null,
  thumbnailImage:
    '//images.ctfassets.net/rj4r6yfcesw5/3Y7zZ25WbeOCU8w84kAEos/b97ff1899cf6f2e4dca0eb2b4f0d5a44/af.png',
  title: 'Arbetsförmedlingen',
}
