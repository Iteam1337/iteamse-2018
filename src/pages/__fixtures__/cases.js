export const cases = [
  {
    location: 'Stockholm',
    shortDescription:
      'Synliggörande av asylsökandes kompetens med digitala verktyg',
    slug: 'vimla',
    thumbnailImage:
      '//images.ctfassets.net/rj4r6yfcesw5/68OlBclfQAGMcEUsQgcIUY/9bdf52bb8a9a17be06e11ef3fa32d3de/vimla-case-phone.png',
    title: 'Vimla',
    __typename: 'Case',
  },
  {
    location: 'Stockholm',
    shortDescription:
      'Synliggörande av asylsökandes kompetens med digitala verktyg',
    slug: 'tsab',
    thumbnailImage:
      '//images.ctfassets.net/rj4r6yfcesw5/UGucAeFycSqIGw0CAc0ys/3d58300261bcff64dba751de9228a150/tsab-case-phone.png',
    title: 'Taxi Stockholm',
    __typename: 'Case',
  },
  {
    location: 'Stockholm',
    shortDescription:
      'Synliggörande av asylsökandes kompetens med digitala verktyg',
    slug: 'seb',
    thumbnailImage: null,
    title: 'SEB',
    __typename: 'Case',
  },
  {
    location: 'Stockholm',
    shortDescription:
      'Synliggörande av asylsökandes kompetens med digitala verktyg',
    slug: 'arbetsformedlingen',
    thumbnailImage:
      '//images.ctfassets.net/rj4r6yfcesw5/3Y7zZ25WbeOCU8w84kAEos/b97ff1899cf6f2e4dca0eb2b4f0d5a44/af.png',
    title: 'Arbetsförmedlingen',
    __typename: 'Case',
  },
]
