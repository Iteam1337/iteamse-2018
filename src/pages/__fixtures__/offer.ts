// tslint:disable:object-literal-sort-keys
export const offers = [
  {
    offerIntroImage:
      '//images.ctfassets.net/rj4r6yfcesw5/5n7J4mzx3UWam0YwgsUQm/8e00d12d6e80eba927b2a313fb250d87/1.png',
    offerTitle: 'Kod',
    offerIntroText:
      'Systemutveckling, automatisering, artificiell intelligens, maskininlärning. Vi har alla redskap du behöver för att bygga vad du vill. Det du vill. Och vet du inte vad du behöver så hjälper vi dig med det med. ',
    offerLead:
      'Aeneanurutrum, erat at tempus porttitor, mi eros vestibulum magna, eget semper mauris ipsum a nisi. Interdum et malesuada fames ac ante ipsum primis in faucibus. Class aptent taciti sociosqu ad litora torquent per conubia nostra, per inceptos himenaeos. Cras quis odio vel turpis pellentesque finibus. Pellentesque cursus diam ac ligula placerat volutpat. Vivamus dictum augue eu rutrum efficitur. Suspendisse ut pellentesque ante. ',
    offerUspOneTitle: 'Utveckling av pilot',
    offerUspOneText:
      'Aeneanurutrum, erat at tempus porttitor, mi eros vestibulum magna, eget semper mauris ipsum a nisi.',
    offerUspTwoTitle: 'API',
    offerUspTwoText:
      'Aeneanurutrum, erat at tempus porttitor, mi eros vestibulum magna, eget semper mauris ipsum a nisi.',
    offerUspThreeTitle: 'Mobila applikationer',
    offerUspThreeText:
      'Aeneanurutrum, erat at tempus porttitor, mi eros vestibulum magna, eget semper mauris ipsum a nisi.',
    offerUspFourTitle: 'AI-lösningar',
    offerUspFourText:
      'Aeneanurutrum, erat at tempus porttitor, mi eros vestibulum magna, eget semper mauris ipsum a nisi\n',
    offerIllustrationImage:
      '//images.ctfassets.net/rj4r6yfcesw5/5SAhc8YuL6OaQQsC6miokK/7c09d594e44ffa47781bf1bdb21aae7f/illustrations-group-2.png',
    offerOrder: 1,
    __typename: 'Offer',
  },
]
