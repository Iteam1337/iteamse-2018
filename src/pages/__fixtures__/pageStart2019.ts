export const pageStart2019 = {
  __typename: 'PageStart2019',
  ctaButtonText: 'Kontakta oss',
  ctaText: 'Hör av dig till oss så pratar vi vidare.',
  ctaTitle: 'Nyfiken på vad vi kan åstadkomma tillsammans?',
  headerLead: `Läs mer om våra [erbjudanden](/erbjudanden "Erbjudanden") eller se våra [case](/case "Våra case")`,
  headerText1:
    'Hej! Vi är Iteam, en utvecklingbyrå som digitaliserar företag genom kod, kultur och strategi.',
  ourMethodText:
    'Vi har varit verksamma som digital innovationsbyrå i över 20 år. Så vi vet att glädje,  teknisk spetskompetens och strategiska perspektiv är viktigt. Men det är när personer med olika kompetenser och varierande bakgrunder samarbetar i tvärfunktionella team som innovation verkligen sker. Det är därför som jobbar vi alltid i team tillsammans med våra kunder. ',
  ourMethodTitle: 'Vår metod',
  team: ['hrn', 'mrs', 'jgn'],
  weAreText:
    'Vi är en digital utvecklingsbyrå med kontor i Stockholm och Göteborg. Totalt är vi är 35 utvecklare, värdecoacher,  UX-designers och stateger som drivs av tre grundläggande värderingar - att göra något bra, att ha kul och att skapa värde. \n\nVi kombinerar strategi, kod och kultur för att ta fram digitala lösningar som möter användarnas behov. Vi utvecklar inte bara appar. Vi tar ett helhetligt perspektiv på din affär för att skapa förändring på riktigt. Vi leder förändringsarbetet och hjälper er att digitalisera er affär.\n\n',
  weAreTitle: 'Vi är Iteam',
  weOfferText:
    'Med ett långsiktigt engagemang hjälper våra kunder att navigera i den digitala världen. Vi omvandlar visioner till verkliget, utmanar föråldrade metoder och tillsammans med er utvecklar vi digitala tjänster som era kunder älskar att använda.',
  weOfferTitle: 'Vi erbjuder',
}
