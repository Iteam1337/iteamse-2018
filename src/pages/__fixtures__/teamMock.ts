import { TEAM_QUERY } from '../../components/Team/Team'
import { teamMembers } from './teamMembers'

export const TeamQueryMock = {
  request: {
    query: TEAM_QUERY,
  },
  result: {
    data: {
      teamMembers,
    },
  },
}
