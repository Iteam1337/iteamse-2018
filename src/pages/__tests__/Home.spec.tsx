import React from 'react'
import { render, wait } from 'react-testing-library'
import MockedQuery from '../../utils/test-utils/MockedQuery'
import { pageStart2019 } from '../__fixtures__/pageStart2019'
import { TeamQueryMock } from '../__fixtures__/teamMock'
import Home, { HOME_PAGE_QUERY } from '../Home'

const mocks = [
  {
    request: {
      query: HOME_PAGE_QUERY,
    },
    result: {
      data: {
        pageStart2019,
      },
    },
  },
  TeamQueryMock,
]

describe('components/Home', () => {
  it('renders Home', async () => {
    const { container } = render(
      <MockedQuery mocks={mocks}>
        <Home />
      </MockedQuery>
    )

    await wait()

    expect(container).toMatchSnapshot()
  })
})
