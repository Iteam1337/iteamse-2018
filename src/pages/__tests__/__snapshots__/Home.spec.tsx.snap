// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/Home renders Home 1`] = `
<div>
  <div>
    <div
      class="sc-jtRfpW bHDxxY"
    >
      <div
        class="sc-kTUwUJ gOkXip"
      >
        <div
          class="sc-cmTdod cbLNuN"
        >
          <a
            class="sc-jwKygS ccXyBf"
          >
            <img
              class="sc-btzYZH gzqeJw"
              data-testid="logo"
              path="test-file-stub"
            />
          </a>
          <div
            class="sc-lhVmIH gEAOxd"
          >
            <a
              class="sc-elJkPf dEOAlI"
              aria-current="false"
            >
              Case
            </a>
            <a
              class="sc-elJkPf dEOAlI"
              aria-current="false"
            >
              AI
            </a>
            <a
              class="sc-elJkPf dEOAlI"
              aria-current="false"
            >
              Metod
            </a>
            <a
              class="sc-elJkPf dEOAlI"
              aria-current="false"
            >
              Medarbetare
            </a>
            <a
              class="sc-elJkPf dEOAlI"
              aria-current="false"
            >
              Karriär
            </a>
            <a
              class="sc-elJkPf dEOAlI"
              aria-current="false"
            >
              Om
            </a>
            <div
              class="sc-bYSBpT BnQkq"
            />
          </div>
        </div>
        <div
          class="sc-dqBHgY bymaoW"
        >
          <h1
            class="sc-gxMtzJ eTxveX"
          >
            <span>
              Vi digitaliserar företag och organisationer
            </span>
          </h1>
          <h1
            class="sc-gxMtzJ eTxveX"
          >
            <span>
              genom strategi, kod och kultur
            </span>
          </h1>
        </div>
      </div>
    </div>
  </div>
  <div
    class="sc-fBuWsC gYhPAl"
  >
    <div
      class="sc-fMiknA dojdZf"
    >
      <img
        class="sc-caSCKo jQCccn"
        src="test-file-stub"
      />
      <div
        class="sc-dVhcbM lmANPA"
      >
        <div
          class="sc-eqIVtm eJqVIb"
        >
          <h1
            class="sc-gZMcBi ctjDhW"
          >
            Kod
          </h1>
          <div
            class="sc-htpNat iEyarI"
          >
            <div>
              <p>
                Systemutveckling, automatisering, artificiell intelligens, maskininlärning. Vi har alla redskap du behöver för att bygga vad du vill. Det du vill. Och vet du inte vad du behöver så hjälper vi dig med det med.
              </p>
            </div>
          </div>
          <img
            class="sc-gisBJw dghrjb"
            src="//images.ctfassets.net/rj4r6yfcesw5/3qnZSfySW4cEGoUo6KCAae/8a656eba15e15b61dacc9fbea38aa4bd/logo_af.png"
          />
          <a
            class="sc-jhAzac eJSshj"
          >
            Läs om hur vi hjälper Arbetsförmedlingen med kod
          </a>
        </div>
        <div
          class="sc-eqIVtm eJqVIb"
        >
          <h1
            class="sc-gZMcBi ctjDhW"
          >
            Strategi
          </h1>
          <div
            class="sc-htpNat iEyarI"
          >
            <div>
              <p>
                Vi digitaliserar Sverige. Pompigt! Men det betyder att vi hjälper dig att leda digitaliseringen istället för att vara den som blir lämnad på efterkälken. Både på den marknad där du verkar, och de marknader du inte ens visste att du borde verka på.
              </p>
            </div>
          </div>
          <img
            class="sc-gisBJw dghrjb"
            src="//images.ctfassets.net/rj4r6yfcesw5/1mv5fxBGLmeyWom4OI2WsI/a4d79cae2076b43aea926db2f1244b52/logo_tsab.png"
          />
          <a
            class="sc-jhAzac eJSshj"
          >
            Läs om hur vi hjälper Taxi Stockholm med strategi
          </a>
        </div>
        <div
          class="sc-eqIVtm eJqVIb"
        >
          <h1
            class="sc-gZMcBi ctjDhW"
          >
            Kultur
          </h1>
          <div
            class="sc-htpNat iEyarI"
          >
            <div>
              <p>
                Ibland ser vi oss som kulturarbetare. Det räcker inte att bygga digitala tjänster åt sina användare. Det är först när hela din organisation tänker, andas och lever digitalt som ni digitaliseras. på riktigt. Vi rustar dig för framtiden helt enkelt.
              </p>
            </div>
          </div>
          <img
            class="sc-gisBJw dghrjb"
            src="//images.ctfassets.net/rj4r6yfcesw5/4D844WR1v20ICIOG4SK0y8/15a008277f5565d9f52fbd5312f6e121/logo_seb.png"
          />
          <a
            class="sc-jhAzac eJSshj"
          >
            Läs om hur vi hjälper SEB med kultur
          </a>
        </div>
      </div>
    </div>
  </div>
  <div
    class="sc-iwsKbI fwFOwW"
  >
    <div
      class="react-reveal"
      style="opacity: 1;"
    >
      <div
        class="sc-bdVaJa fGzlqx"
        data-test="block-"
      >
        <div
          class="sc-bxivhb kgeHJJ"
        >
          <div
            class="sc-ifAKCX cFlEyZ"
          >
            <h3
              class="sc-bwzfXH kgEpTV"
            >
              Utforska möjligheterna med AI
            </h3>
          </div>
          <div
            class="sc-bZQynM hmvJjY"
          >
            <div
              class="sc-fAjcbJ byigni"
            >
              We are offering an Artificial Intelligence package to help you build AI products and services. We have a very competent Data Science team that will help you grow in the growing market of Artificial Intelligence.
            </div>
            <div
              class="sc-fAjcbJ byigni"
            >
              <a
                class="sc-jhAzac eJSshj"
              >
                Läs mer om våra erbjudanden inom AI
              </a>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
